setup-knative-eventing/README.md

```diff
@@ -0,0 +1,23 @@
+# Setup Knative
+
+This action installs Knative Eventing into the current kubectl context.
+
+## Usage
+
+```yaml
+- uses: chainguard-dev/actions/setup-knative-eventing@main
+  with:
+    # Version is the version of Knative Eventing to install.
+    # For example, 1.2.0.
```

Comment:
change 1.2.0 to 1.5.0 to ensure people dont actually use an older version.

Reply:
Done